src/components/Book.js

```diff
@@ -0,0 +1,68 @@
+// Individual Book component
+
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+import { RemoveBookButton } from './Button';
+import '../styles/book.css';
+import progress from '../Assets/progress.png';
+
+const Book = ({
+  id, title, author, category,
+}) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveClick = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+    <div className="book">
+      <div className="book-cover">
+        <div className="book-info">
+          <h6 className="category">
+            {category}
+            Action
+          </h6>
+          <h3 className="title">
+            {title}
+          </h3>
+          <p className="author">
+            {author}
+          </p>
+        </div>
+        <div className="book-actions">
+          <p className="comments">
+            Comments
+          </p>
+          <span className="remove">
+            <RemoveBookButton className="remove-btn" type="button" onClick={handleRemoveClick}>Remove</RemoveBookButton>
+          </span>
+          <p className="edit">
+            Edit
+          </p>
+        </div>
+      </div>
+      <div className="progress">
+        <img src={progress} alt="progress" />
+        <div className="progress-info">
+          <p className="parcent">64%</p>
+          <p className="completed">Completed</p>
+        </div>
+      </div>
+      <div className="chapter">
+        <p className="current">CURRENT CHAPTER</p>
+        <p className="chapter-number">Chapter 17</p>
+        <button type="button" className="update-progress"><p>UPDATE PROGRESS</p></button>
+      </div>
```

Comment:
- [ ] As I mentioned above please stick to the Zaplin design but you missed the horizontal divider between the progress bar and the Current Chapter so kindly add a horizontal divider :+1: check below :point_down: the comparison between your design and Zaplin design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-09-15 18-04-13](https://github.com/samuelkinyanjui/bookstore_add_reducers/assets/45226848/42e57412-feae-46ea-be02-b9d090b4f2dc)

**Zaplin Design**
![Screenshot from 2023-09-15 18-04-04](https://github.com/samuelkinyanjui/bookstore_add_reducers/assets/45226848/3dccbcc9-ed16-4448-9e03-0fe203d912c7)

- [ ] Also if you check your Current Chapter text it does not match with Zaplin design so kindly fix the colour and font style as well :slightly_smiling_face: